DataConnectors/Templates/Connector_REST_API_AzureFunctionApp_template/Template_REST_API_AzureFunction_App_Code/Template_REST_API_Function_App_Python/Template_REST_API_Function_App_Python.py

```diff
@@ -36,7 +37,16 @@ def main(mytimer: func.TimerRequest) -> None:
 customer_id = os.environ['workspaceId'] 
 shared_key = os.envviron['workspaceKey']
 log_type = os.envviron['tableName']
-
+logAnalyticsUri = os.environ['logAnalyticsUri']
+if (not str(logAnalyticsUri)): 
+    logAnalyticsUri = "https://" + customer_id + ".ods.opinsights.azure.com"
+
+pattern = r"https:\/\/([\w\-]+)\.ods\.opinsights\.azure.([a-zA-Z\.]+)$"
+match = re.match(pattern,str(logAnalyticsUri))
+if(not match):
+    logging.error("Invalid Log Analytics Uri.")
```

Comment:
Need to raise an exception instead of logging. If we do that no need to use sys.exit()

raise Exception("Invalid Log Analytics Uri.")